php-fpm support information (#3504)
owncloud/docs: modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc

== PHP-FPM

> Please note that the ownCloud QA team is using solely mod_php for testing. 

Reviewer: Please note that the ownCloud QA team is using solely mod_php for testing.

Note that `mod_php` is used exclusively in the development and QA process of the ownCloud server.

> Therefore in a production environment it is recommended to do the same.

Reviewer: Therefore in a production environment it is recommended to do the same.

It's highly recommended to use `mod_php` in your production environment for optimal performance and stability.

> The ownCloud support team requires the reproduction of any potential cases

Reviewer: The ownCloud support team requires the reproduction of any potential cases
 with mod_php.

Any issues with the ownCloud server have to be reproducible with `mod_php`.

> The ownCloud support team requires the reproduction of any potential cases
> with mod_php.

Reviewer: ```suggestion
Any issues with the ownCloud server have to be reproducible with `mod_php`
```

> Additionally if you are looking to use SAML SSO with Shibboleth, this

Reviewer: Additionally if you are looking to use SAML SSO with Shibboleth, this
 does not work with PHP-FPM.

SAML SSO with Shibboleth **will not work** with `php-fpm`.

> Additionally if you are looking to use SAML SSO with Shibboleth, this
> does not work with PHP-FPM.

Reviewer: ```suggestion
SAML SSO with Shibboleth **will not work** with `php-fpm`.
```
